templates/desktop/features.html

```diff
@@ -41,7 +14,7 @@ <h1>Features</h1>
     </div>
     <div class="col-6 u-vertically-center u-hide--medium u-hide--medium u-hide--small u-align--center">
       {{ image (
-        url="https://assets.ubuntu.com/v1/fa57dc30-desktop-22.04-laptop.png",
+        url="https://assets.ubuntu.com/v1/1930f20c-desktop-xps13-alt.png",
         alt="",
         width="1000",
         height="578",
```

Comment:
The image is distorted, this should do it:
```suggestion
        width="600",
        height="498",
```